<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKTransmissionSettings</class>
 <widget class="QWidget" name="SAKTransmissionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据转发</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="serialPortWidget">
      <attribute name="title">
       <string>串口转发</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="udpWidget">
      <attribute name="title">
       <string>UDP转发</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tcpWidget">
      <attribute name="title">
       <string>TCP转发</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
